meson.build

```diff
@@ -0,0 +1,126 @@
+project('vkBasalt', 'cpp', default_options: ['c_std=c11', 'cpp_std=c++2a'])
+
+vulkan = dependency('vulkan')
+
+add_project_link_arguments(['-lstdc++fs'], language: 'cpp')
```

Comment:
You probably don't need to explicitly add link args for `stdc++` since you set the standard in `project` section.

Reply:
This was still needed here, as clang in particular didn't link against `stdc++fs` automatically. Obviously `stdc++` was linked, but `stdc++fs` was not.

Reply:
Meson should handle it just fine. An by explicitly adding `-lstdc++fs` you break the build on systems without `libstdc++fs.a` present.